backend/scripts/cleanup_lookups/cleanup_lookups.py

```diff
@@ -82,6 +87,62 @@ def cleanup_sector_duplicates(db: SessionLocal) -> None:  # type: ignore
             db.delete(sector)
 
 
+def load_keyword_map() -> Mapping[str, str]:
+    current_folder = pathlib.Path(__file__).parent.absolute()
+    keyword_map_file = current_folder / "keyword_map.csv"
+    with open(keyword_map_file, "r") as keyword_map_file_handle:
+        keyword_map_csv = DictReader(keyword_map_file_handle)
+        keyword_map = {}
+        for row in keyword_map_csv:
+            keyword_map[row["name"]] = row["desired_keyword"]
+
+        return keyword_map
+
+
+def get_clean_keyword_map(db: SessionLocal) -> Mapping[int, int]:  # type: ignore
+    """Gets values for updating document keyword references"""
+    keyword_map = load_keyword_map()
+
+    existing_keywords = set(keyword_map.keys())
+    desired_keywords = set(keyword_map.values())
+    keywords_to_create = desired_keywords - existing_keywords
+
+    name_lookup: Dict[str, int] = {}
+    for k in keywords_to_create:
+        new_keyword = Keyword(name=k, description="Imported by CPR loader")
+        db.add(new_keyword)
+        db.flush()
+        db.refresh(new_keyword)
+        name_lookup[k] = int(new_keyword.id)  # type: ignore
```

Comment:
My preference would be to add this as a function to the class `Keyword` - in fact what do you think about putting all the db operations for Keywords on the keyword class?

Reply:
After talking about this I think we agreed to push the discussion of these functions to a later date.